src/pages/blog/discovered-a-cool-javascript-property/index.mdx

```diff
@@ -1,14 +1,14 @@
 ---
 title: "Discovered a cool javascript property"
-description: ""
+description: "Yesterday I was playing around with javascript trying to find the cleanest way to implement callbacks in functions. Primarily I wanted a readable way to make certain the last argument passed is a callback, withut having to rely on knowing how many arguments there are and so on. Instead I stumbled upon a pretty cool feature of how javascript handles function arguments."
```

Comment:
Let’s retitle this article to `A cool JavaScript property you never noticed`

and change `yesterday` to “a few years ago” in both the article and text.

No need to move the article, can keep the same URL.